### PR TITLE
added utkarsh.json

### PR DESCRIPTION
**Pull request by utkarshkrsingh**
- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable. 
- [x] You're not using Vercel.
- [x] The CNAME record doesn't contain `https://` or `/`. 
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/utkarsh.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "repo": "https://github.com/utkarshkrsingh/website",
+  "owner": {
+    "username": "Utkarshkrsingh",
+    "email": "utkarshkrsingh@proton.me"
+  },
+  "record": {
+    "CNAME": "utkarshkrsingh.github.io"
```
Comment (21Z) [inline domains/utkarsh.json:8]: I except your apology, but I can't seem to access the website.
Gives 404, and can't find a website on your repo either.
if this is the correct record, could you send the URL?

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by utkarshkrsingh** [comment]
My apology,
I am uploaded the files in website repository as well as started the utkarshkrsingh.github.io.
I am really sorry I forgot this and wasted your time. Thank you for informing me.

File: domains/utkarsh.json (hunk 1 above)
Comment (utkarshkrsingh) [reply domains/utkarsh.json:8]: My apology,
I am uploaded the files in website repository as well as started the utkarshkrsingh.github.io.
I am really sorry I forgot this and wasted your time. Thank you for informing me.

Comment (utkarshkrsingh) [reply domains/utkarsh.json:8]: Doing it contribution for first time, that's why a little confused and excited too.

**Comment by 21Z** [comment]
You need to chill with your apologies bro, don't worry you'll PR will be merged by a maintainer soon

**Comment by utkarshkrsingh** [comment]
Sure. Thankx 😃

Sent from Proton Mail Android

-------- Original Message --------

**Review by iostpa — APPROVED** [review]
Seems fine to me now

**Pull request merged**